features/works/external_works.feature

```diff
@@ -0,0 +1,26 @@
+@works
+
+Feature: External works are not hosted on the Archive
```

Comment:
This should live in features/bookmarks/bookmark_external.feature along with similar tests.